File: cpp/if else/marketprice.cpp
/*according the marked price from the user and calculate the net amount as(market price-discount) to pay according to following criteria
>10000 20%
>7000 and <=10000      15%
<=7000      10%*/

#include <iostream>
using namespace std;

int main() {
    float marketPrice, discount = 0.0, netAmount;

    cout << "Enter the market price: ";
    cin >> marketPrice;

    if (marketPrice > 10000) {
        discount = marketPrice * 0.20;
    } 
    else if (marketPrice > 7000 && marketPrice <= 10000) {
        discount = marketPrice * 0.15;
    } 
    else {
        discount = marketPrice * 0.10;
    }

    netAmount = marketPrice - discount;

    cout << "Net amount to pay: Rs " << netAmount << endl;

    return 0;
}

File: java/if else/marketprice.java
/*#according the marked price from the user and calculate the net amount as(market price-discount) to pay according to following criteria
#>10000 20%
#>7000 and <=10000      15%
#<=7000      10%*/

import java.util.Scanner;

class marketprice{
    public static void main(String args[]){
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter the marked price:");
        int price=sc.nextInt();

        double netamount=0;

        if(price>10000){
            netamount=price-(price*0.2);
        }
        else if(price>7000 && price<=10000){
            netamount=price-(price*0.15);
        }
        else{
            netamount=price-(price*0.1);
        }

        System.out.println("The net amount to pay is: Rs."+netamount);
    }
}


